nvflare/private/fed/client/client_run_manager.py

```diff
@@ -128,10 +128,10 @@ def new_context(self) -> FLContext:
 
     def send_task_result(self, result: Shareable, fl_ctx: FLContext) -> bool:
         push_result = self.client.push_results(result, fl_ctx)  # push task execution results
-        if push_result[0] == CellReturnCode.OK:
+        if push_result == CellReturnCode.OK:
             return True
         else:
-            return False
+            self.logger.error(f"failed to send task result: {push_result}")
```

Comment:
Do we need to `return False` here?

Reply:
Yes we should, even though Python would return None for us (which is almost equivalent to False). Will fix.

Reply:
Fixed.